simpleocr/files.py

```diff
@@ -10,21 +10,8 @@
 GROUND_EXTENSIONS_DEFAULT = GROUND_EXTENSIONS[0]
 
 
-def open_img(path_or_name):
```

Comment:
IMHO, having a big *anything* (function, method, class, list of imports, file) is not *wrong* by itself, but usually indicates something is wrong at the design level. YMMV.

A good reason to keep it separate is because, likely, if we ever open ground files by themselves, we'll want to avail of the same mechanism (although at that point we'll have to further decompose it). It can be thought as a generic operation on file paths, as opposed to something related to images